projects/ngx-datatable/src/lib/components/datatable.component.html

```diff
@@ -75,8 +74,12 @@
       [rowDragEvents]="rowDragEvents"
       [rowDefTemplate]="rowDefTemplate"
     >
-      <ng-content select="[loading-indicator]" ngProjectAs="[loading-indicator]"></ng-content>
-      <ng-content select="[empty-content]" ngProjectAs="[empty-content]"></ng-content>
+      <ng-content select="[loading-indicator]" ngProjectAs="[loading-indicator]">
+        <datatable-progress></datatable-progress>
+      </ng-content>
```

Comment:
I am uncertain where the fallback should actually be.  
My first assumption was, that this should be part of the component actually showing this --> so the body component

@timowolf your opinion?

Reply:
If we move it to the inner component then it will not work because of the nested `ng-content` structure here.

Reply:
this makes the decission easy